server/src/main/java/io/deephaven/server/table/ExportedTableUpdateListener.java

```diff
@@ -133,6 +134,11 @@ private synchronized void onNewTableExport(final Ticket ticket, final int export
             sendUpdateMessage(ticket, table.size(), null);
             return;
         }
+        if (table.isFailed()) {
```

Comment:
This is nice, but doesn't really completely do the job. We want to also handle races in the snapshot. Talked about adding more specific exception types to two layers of the stack.